openprescribing/openprescribing/settings/base.py

```diff
@@ -3,9 +3,19 @@
 from sys import path
 from django.core.exceptions import ImproperlyConfigured
 from common import utils
+import warnings
 
 import sys
 
+# This `cryptography` warning is making our cronjobs chatty
+warnings.filterwarnings(
+    "ignore", "Python 3.5 support will be dropped in the next release"
+)
+if sys.version_info >= (3, 6):
+    warnings.warn(
+        "Warning supression on line above is now redundant and should be removed"
```

Comment:
nice!

Reply:
It's a bit meta, isn't it? A warning about a thing which turns off a warning. I'm just keen to work against the inevitable accumulation of cruft that all projects develop.